minorminer/utils/embeddings.py

```diff
@@ -307,20 +328,19 @@ def embedding_feasibility_filter(
             return True
 
 
-def raster_breadth_subgraph_upper_bound(T: nx.Graph = None) -> int:
-    """Determines a raster breadth upper bound for subgraph embedding.
+def graph_rows_upper_bound(T: nx.Graph = None) -> int:
```

Comment:
what's a "graph rows"?

Reply:
The parameter 'm' of Chimera, Pegasus and Zephyr graphs is equal to G.graph['rows'] and is described in documentation as number of rows. The raster breadth is the value of m describing the subgraph. I think we can improve the documentation, but I'm struggling for a better choice than graph rows.

Reply:
I will improve the document if you two don't have any specific suggestions.
We could try graph_rows_m (to make a connection with the shape parameter m)?
sublattice_rows ?
sublattice_rows_m ?
sublattice_dimension?